### ISSUE TITLE
'Kill all table rows' webui e2e test in 'Kill all experiments' step in 'Clean Up' spec times out when additional table columns are added

### ISSUE THREAD

**Issue by EmilyBonar** [determined-ai/determined#3477]
The webui-e2e test times out on CircleCI at the step listed in the title but passes locally.

**Comment by hamidzr** [comment]
https://github.com/determined-ai/determined/pull/3480/checks?check_run_id=4901512514

more examples of this happening https://app.circleci.com/pipelines/github/determined-ai/determined/18404/workflows/5d70c3b8-b14a-44d8-8826-2ff8d31ebdec/jobs/566595

**Comment by hamidzr** [comment]
```
repro steps: 
  - add a few columns: test PR HERE

- gauge fails to output anything and fails to timeout gracefully
- we cannot reproduce locally
  - slower, more similar test env. maybe using circleci-cli

INCLUDE what we tried
```

**Comment by hamidzr** [comment]
@EmilyBonar could you please write down what we tried and observed?

**Comment by EmilyBonar** [comment]
We noticed that the following step timed out on CircleCI but not locally.
```
@Step('<action> all table rows')
  public async actionOnAllTableRows(action: string) {
    await t.click(BATCH_ACTION_TEXT);
    // Wait for the dropdown animation to finish
    await sleep(500);
    await t.click(action, t.within(t.$('.ant-select-dropdown')));
    // Wait for the modal to animate in.
    await t.waitFor(async () => !(await t.$('.ant-modal.zoom-enter').exists()));
    await t.click(t.button(action, t.within(t.$('.ant-modal-body'))));
    // Wait for the modal to animate away
    await t.waitFor(async () => !(await t.$('.ant-modal.zoom-leave').exists()));
  }
```
When I added console.log commands before and after every step, it seemed that the specific line of code that was stalling was the first one (`await t.click(BATCH_ACTION_TEXT);`). We tried using Taiko to wait for the `BATCH_ACTION_TEXT` element to exist, be visible, and not be disabled. This had no effect. We tried adding verification of those qualities to the previous step in the test section. The verification succeeded, but had no effect on the timeout. We tried removing the columns that were added to the Experiment List. This worked to eliminate the timeout, but the fact that this error was seen on PRs with no WebUI changes at all proves that the test was flaky to begin with and adding the columns just exacerbated the issue. We tried scrolling the table to the left, in case some element was off screen. This had no effect.
